### PR TITLE
Fix site metadata for French

### LINKED ISSUE #156: [Language] Fix French Page
Fix the metadata of the French page

### PR DESCRIPTION
**Pull request by GindaChen**
Fix #156

### CODE REVIEW

**Comment by GindaChen** [comment]
(I think the GithubDesktop has a bug -- if I switch branch, the most recent PR will be merged ...

File: site_config/site.js [hunk 1]
```diff
@@ -88,7 +88,7 @@ export default {
                 },
             ],
         },
-        copyright: 'Copyright © 2020 Wuahan2020',
+        copyright: 'Copyright © 2020 Wuhan2020',
```
Comment (chenrui333) [inline site_config/site.js:91]: good catch!

File: site_config/site.js [hunk 2]
```diff
@@ -389,80 +389,80 @@ export default {
...
+                text: 'PRODUCT',
                 link: '/fr-fr/blog/download.html',
             },
             {
-              key: 'mis',
-              text: '新型冠状病毒防疫信息平台',
-              link: 'https://wuhan2020.kaiyuanshe.cn',
+                key: 'mis',
+                text: '2019-NCOV INFORMATION PLATFORM',
+                link: 'https://wuhan2020.kaiyuanshe.cn',
```
Comment (chenrui333) [inline site_config/site.js:424]: 这里是有问题的

File: site_config/site.js [hunk 3]
```diff
@@ -389,80 +389,80 @@ export default {
...
                 },
                 {
-                    text: '社区',
+                    text: 'Community',
                     link: '/fr-fr/community/index.html',
                 },
             ],
         },
-        copyright: 'Copyright © 2020 Wuhan2020',
+        copyright: 'Copyright © 2020 Wuahan2020',
```
Comment (chenrui333) [inline site_config/site.js:465]: typo in here.

Comment (GindaChen) [reply site_config/site.js:465]: Fix soon

File: site_config/site.js [hunk 4]
```diff
@@ -389,80 +389,80 @@ export default {
...
+            title: 'Vision',
+            content: 'Unite as one and drive out the plague!',
         },
         documentation: {
-            title: '文档',
+            title: 'Documentation',
             list: [
                 {
-                    text: 'Wuhan2020 项目是干什么的？',
+                    text: 'What is Wuhan2020?',
```
Comment (chenrui333) [inline site_config/site.js:435]: 你这里应该是法语翻译。

File: site_config/site.js [hunk 5]
```diff
@@ -389,80 +389,80 @@ export default {
...
                     link: '/fr-fr/docs/overview/what-is-wuhan2020.html',
                 },
                 {
-                    text: '快速开始',
+                    text: 'Quick Start',
```
Comment (chenrui333) [inline site_config/site.js:439]: 你这里应该是法语翻译。

File: site_config/site.js (hunk 2 above)
Comment (GindaChen) [reply site_config/site.js:424]: I haven't understand the problem yet...  What is the problem here?

Comment (chenrui333) [reply site_config/site.js:424]: code style issue, indentation problem

**Pull request merged**